src/main/java/carpet/CarpetSettings.java

```diff
@@ -137,7 +138,7 @@ private static void set_defaults()
   //                              .extraInfo("Also prevents rotations upon placement of dispensers and furnaces","when placed into a world by commands"),
   /////rule("optimizedTNT",          "tnt", "TNT causes less lag when exploding in the same spot and in liquids"),
   /////rule("huskSpawningInTemples", "experimental feature", "Only husks spawn in desert temples"),
-  /////rule("shulkerSpawningInEndCities", "feature experimental", "Shulkers will respawn in end cities"),
```

Comment:
this was marked with ///// meaning - do not port...
I heard there were performance issues with this one.

Reply:
Ohh okay.
Performance problems, hmm, I'll have a close look at it as I'm using it anyway.

Reply:
The performance issues were likely due to the fact that `recreateStructures` was called in the end, even when the rule was disabled. I don't know whether this would affect 1.13 due to the world gen rewrite.